packages/eds-core-react/src/components/Button/ToggleButton/ToggleButton.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  HTMLAttributes,
+  cloneElement,
+  Children as ReactChildren,
+  forwardRef,
+  ReactElement,
+  useState,
+  useEffect,
+} from 'react'
+import { Button, ButtonProps, ButtonGroupProps } from '..'
+
+export type ToggleButtonProps = {
+  /** Multiple */
+  multiple?: boolean
+  /** Array of selected indexses. */
+  selected?: number[]
```

Comment:
Should we maybe call this `selectedIndexes` instead? What do you think @oddvernes ?